src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ComboBoxTests.cs

```diff
@@ -617,6 +617,120 @@ public void ComboBox_DataSource_SetWithHandler_CallsDataSourceChanged()
         Assert.Equal(0, displayMemberCallCount);
     }
 
+    public class CustomComboBox : ComboBox
+    {
+        // Make methods private to improve encapsulation
+        private void RaiseOnDrawItem(DrawItemEventArgs e) => base.OnDrawItem(e);
+
+        private void TriggerDoubleClick() => base.OnDoubleClick(EventArgs.Empty);
+
+        // Public methods to trigger the private methods for testing purposes
+        public void TestRaiseOnDrawItem(DrawItemEventArgs e) => RaiseOnDrawItem(e);
+
+        public void TestTriggerDoubleClick() => TriggerDoubleClick();
+    }
+
+        [WinFormsFact]
+        public void ComboBox_DrawItem_AddHandler_ShouldCallHandler()
+        {
+            using CustomComboBox control = new();
```

Comment:
Please fix indentation. New tests added past this point are indented in once too much
```suggestion
    }

    [WinFormsFact]
    public void ComboBox_DrawItem_AddHandler_ShouldCallHandler()
    {
        using CustomComboBox control = new();
```